### PR TITLE
Emotes now check if there is actually somebody to hear it in earshot

### PR DESCRIPTION
**Pull request by MadmanMartian**
If the person making the emote doesn't have a mind or ckey, it checks in proximity of the user, to see if anyone can hear or see the emote.

### CODE REVIEW

File: code/datums/emotes.dm [hunk 1]
```diff
@@ -141,7 +141,14 @@
 		return FALSE
 	if((isvox(user) || isskelevox(user)) && voxemote == FALSE)
 		return FALSE
-
+	if(!user.client && user.ckey == null) //Auto emote, like a monkey or corgi
+		var/someone_in_earshot=0
+		for(var/mob/M in get_hearers_in_view(7, user)) //See if anyone is in earshot
```
Comment (Exxion) [inline code/datums/emotes.dm]: Should be `world.view` rather than `7`

**Comment by SonixApache** [comment]
if a corgi barks in the woods, and there's nobody there to hear it, does it make a sound?

**Comment by SirSkeleto** [comment]
this work for russians too? they clog up deadchat sometimes

**Comment by MadmanMartian** [comment]
They're actively saying something, so afraid not

File: code/datums/emotes.dm (hunk 1 above)
Comment (Intigracy) [inline code/datums/emotes.dm]:
```suggestion
		for(var/mob/M in get_hearers_in_view(world.view, user)) //See if anyone is in earshot
```

**Review by Intigracy — DISMISSED** [review]

**Pull request merged**